bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/HToZATo2L2B_1000p00_200p00_0p50_bbH4F_TuneCP5_13TeV-amcatnloFXFX-pythia8/HToZATo2L2B_1000p00_200p00_0p50_bbH4F_TuneCP5_13TeV-amcatnloFXFX-pythia8_run_card.dat

```diff
@@ -0,0 +1,196 @@
+  -1.0	= rw_fscale_down  ! lower bound for fact scale variations
+  -1.0	= rw_rscale_down  ! upper bound for fact scale variations
+  -1.0	= rw_fscale_up   ! upper bound for fact scale variations
+  -1.0	= rw_rscale_up   ! upper bound for ren scale variations
+  325501   = PDF_set_min      ! First of the error PDF sets 
+  325600   = PDF_set_max      ! Last of the error PDF sets
+#************************************************************************
+## Renormalization and factorization scales (advanced and NLO options)  *
+##***********************************************************************
+  False	= fixed_qes_scale     ! if .true. use fixed Ellis-Sexton scale
+  91.118	= qes_ref_fixed   ! fixed Ellis-Sexton reference scale
+  1.0	= mur_over_ref ! ratio of current muR over reference muR
+  1.0	= qes_over_ref ! ratio of current QES over reference QES
+#***********************************************************************
+# Store reweight information in the LHE file for off-line model-       *
+# parameter reweighting at NLO+PS accuracy                             *
+#***********************************************************************
+  True	= store_rwgt_info ! Store info for reweighting in LHE file
+#***********************************************************************
+# ickkw parameter:                                                     *
+#   0: No merging                                                      *
+#   3: FxFx Merging - WARNING! Applies merging only at the hard-event  *
+#      level. After showering an MLM-type merging should be applied as *
+#      well. See http://amcatnlo.cern.ch/FxFx_merging.htm for details. *
+#   4: UNLOPS merging (with pythia8 only). No interface from within    *
+#      MG5_aMC available, but available in Pythia8.                    *
+#  -1: NNLL+NLO jet-veto computation. See arxiv:1412.8408 [hep-ph].    *
+#***********************************************************************
+  3	= ickkw 
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma). Determines which resonances are      *
+# written in the LHE event file                                        *
+#***********************************************************************
+  15.0	= bwcutoff 
+#***********************************************************************
+# Cuts on the jets. Jet clustering is performed by FastJet.            *
+#  - When matching to a parton shower, these generation cuts should be *
+#    considerably softer than the analysis cuts.                       *
+#  - More specific cuts can be specified in SubProcesses/cuts.f        *
+#***********************************************************************
+  1.0	= jetalgo ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+  1.0	= jetradius ! The radius parameter for the jet algorithm
+  20.	= ptj ! Min jet transverse momentum
```

Comment:
Is this soft enough compared to cut at analysis level?

Reply:
maybe not sure ! i set ptj = 0.01 as suggested [here](https://twiki.cern.ch/twiki/bin/view/CMS/QuickGuideMadGraph5aMCatNLO#Troubleshooting_and_Suggestions) !
at the analysis level we cut on j.pt > 20.  for resolved jets and j.pt> 200. for the boosted one